Resources/Prototypes/secret_weights.yml

```diff
@@ -4,7 +4,5 @@
     Nukeops: 0.20
     Traitor: 0.60
     Zombie: 0.04
```

Comment:
```suggestion
    Zombie: 0.05
```
At least divide up the chance nicely. A 4 and 11% chance is annoying me way too much 😄

Reply:
Increasing zombie chance??? Evil man, you are...
Pushed requested changes.

Reply:
The 1 additional percent of zombies was already in zombieteors, so this keeps is the same actually.